tests/CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ else()
   FetchContent_Declare(
     Catch2
     GIT_REPOSITORY https://github.com/catchorg/Catch2.git
-    GIT_TAG        037ddbc75cc5e58b93cf5a010a94b32333ad824d
+    GIT_TAG        a4e4e82474f894d56cbe86e13a2d23bb1b46744d
```

Comment:
This is pretty arbitrary, other than fixing the usual MINSIGSTKSZ issue by going to a newer catch2 version.